web/src/components/dashboard/ElevatorViolationsCard.tsx

```diff
@@ -6,21 +8,30 @@ import { ReactElement } from "react";
 
 export const ElevatorViolationsCard = (): ReactElement => {
   const config = getMergedConfig();
+
+  const componentOnLeftSide = (
```

Comment:
Can we rename this to `violationMessage`? makes sense for the prop to be `stackOnLeft` but within this component I think we can be more specific

Reply:
updatet